pytest/test_bem_with_quadratures.py

```diff
@@ -1,11 +1,19 @@
 #!/usr/bin/env python
 # coding: utf-8
 
+import pytest
 import numpy as np
 import xarray as xr
 import capytaine as cpt
+
 import quadpy
+from packaging import version
+quadpy_version = version.parse(quadpy.__version__)
+if not quadpy_version <= version.parse('0.14'):
```

Comment:
Maybe we should test both if quadpy is installed and if the version is correct?

Reply:
Right, so we need an `and` statement

Reply:
0e9d111